src/components/structures/LeftPanel.tsx

```diff
@@ -309,6 +309,24 @@ export default class LeftPanel extends React.Component<IProps, IState> {
                 }
                 break;
         }
+        const navAction = getKeyBindingsManager().getNavigationAction(ev);
+        switch (navAction) {
+            case KeyBindingAction.NextLandmark:
+                ev.stopPropagation();
+                ev.preventDefault();
+                // The next landmark is the selected room or the room first room, if none selected.
+                (
+                    document.querySelector<HTMLElement>(".mx_RoomTile_selected") ||
+                    document.querySelector<HTMLElement>(".mx_RoomTile")
+                )?.focus();
```

Comment:
Any chance we could have a single class/something which is responsible for the overarching order of landmarks in this navigation so you don't have to read a bunch of files to maintain it. Like an enum for the various landmarks and a function with signature `navigateLandmark(currentLandmark: EnumType, backward: boolean): void` or similar